_includes/sidebar.html

```diff
@@ -38,11 +38,11 @@
         <h3>Tutorials</h3>
         <nav>
             <ul class="list-unstyled">
+                <li><a href="/tutorials/">Overview</a></li>
+                <li><a href="/geomesa-deployment/">GeoMesa Deployment</a></li>
                 <li><a href="/geomesa-quickstart/">GeoMesa Quick Start</a></li>
-                {% for tutorial in site.posts reversed limit: 3 %}
-                    <li><a href="{{ tutorial.url }}">{{ tutorial.title }}</a></li>
-                {% endfor %}
-                <li><a href="/tutorials/">Additional Tutorials...</a></li>
+                <li><a href="/geomesa-kafka-quickstart/">Geomesa Kafka Quick Start</a></li>
+                <li><a href="/tutorials/all-by-date/">All by Date...</a></li>
```

Comment:
I think we should get rid of All "by date". We should just have an all section and the section should just be a set of bullet points of the tutorial titles:

-Ingest & Export with GeoMesa Tools
- Management with GeoMesa Tools
- etc

Thoughts?